Project.toml

```diff
@@ -8,7 +8,7 @@ Libdl = "8f399da3-3557-5675-b5ff-fb832c97cbdb"
 MathOptInterface = "b8f27783-ece8-5eb3-8dc8-9495eed66fee"
 
 [compat]
-CEnum = "^0.1.0"
+CEnum = "^0.1.0, 0.2"
```

Comment:
I'm not sure what this means. Did you require `CEnum` in version 0.2?
I don't think it's compatible with our generated wrapper code.

Reply:
It need version 0.2 or later to install SCIP interface version 7.4 above 
Current CEnum version is 0.2
New SCIP interface version not installed without 0.2

Reply:
I don't understand. The newest version of SCIP.jl is 0.9.2 and the newest version of SCIP is 6.0.2. So what is this 7.4 refering to?

I did not try Julia 1.3.0 yet, so maybe it's related to that.

Reply:
I just tried to `add SCIP` to a fresh Julia 1.3.0 installation, which worked just fine. It installed `CEnum 0.1.2` as a dependency and the tests passed. Maybe you have some other package that requires `CEnum 0.2`?

Also, see #144 for failing tests with `CEnum 0.2`. Your changes probably fix those.

So, while it does make sense to upgrade `CEnum` at some point, I would prefer to do it "properly", and regenerate the C wrappers with a new `Clang` etc.